sdk/ml/azure-ai-ml/azure/ai/ml/entities/_load_functions.py

```diff
@@ -452,6 +453,37 @@ def load_data(
     return load_common(Data, source, relative_origin, **kwargs)
 
 
+def load_data_import(
```

Comment:
I don't see sdk experience in this spec: https://github.com/Azure/azureml_run_specification/blob/master/specs/managed-data-asset/data-import-experience1.md

Could you point me to it?

Reply:
https://github.com/Azure/azureml_run_specification/blob/master/specs/python-sdk/import-data-python.md